docker/errors.py

```diff
@@ -127,8 +127,14 @@ def __init__(self, container, exit_status, command, image, stderr):
         self.command = command
         self.image = image
         self.stderr = stderr
-        msg = ("Command '{}' in image '{}' returned non-zero exit status {}: "
-               "{}").format(command, image, exit_status, stderr)
+
+        if stderr is None:
+            msg = ("Command '{}' in image '{}' returned non-zero exit "
+                   "status {}").format(command, image, exit_status, stderr)
+        else:
+            msg = ("Command '{}' in image '{}' returned non-zero exit "
+                   "status {}: {}").format(command, image, exit_status, stderr)
```

Comment:
This change seems unrelated to the issue at hand. Did you run into an issue with this specifically? `format` should work fine with `None` values.